src/Articles.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from "react"
+import getArticles from "./api"
+
+const Articles = ()=>{
+
+    const[articles, setArticles] = useState([])
+
+    useEffect(() => {
+    getArticles().then((articleData) => {
+        setArticles(articleData.articles);
+    })
+    }, []);
+
```

Comment:
Please add a loading state and some visual feedback for the user that the data is loading